src/main/java/org/prebid/server/spring/config/AnalyticsConfiguration.java

```diff
@@ -52,6 +58,64 @@ LogAnalyticsReporter logAnalyticsReporter(JacksonMapper mapper) {
         return new LogAnalyticsReporter(mapper);
     }
 
+    @Configuration
+    @ConditionalOnProperty(prefix = "analytics.greenbids", name = "enabled", havingValue = "true")
+    public static class GreenbidsAnalyticsConfiguration {
+
+        @Bean
+        GreenbidsAnalyticsReporter greenbidsAnalyticsReporter(
+                GreenbidsAnalyticsConfigurationProperties greenbidsAnalyticsConfigurationProperties,
+                GreenbidsJacksonMapper jacksonMapper,
+                HttpClient httpClient,
+                Clock clock,
+                PrebidVersionProvider prebidVersionProvider) {
+            return new GreenbidsAnalyticsReporter(
+                    greenbidsAnalyticsConfigurationProperties.toComponentProperties(),
+                    jacksonMapper,
+                    httpClient,
+                    clock,
+                    prebidVersionProvider);
+        }
+
+        @Bean
+        public ObjectMapper greenbidsObjectMapper() {
+            return new ObjectMapper();
+        }
+
+        @Bean
+        public GreenbidsJacksonMapper greenbidsJacksonMapper(ObjectMapper greenbidsObjectMapper) {
+            return new GreenbidsJacksonMapper(greenbidsObjectMapper);
+        }
```

Comment:
do you need a greenbidsObjectMapper separately from the greenbidsJacksonMapper? If no, please do not declare the object mapper bean